app/controllers/rents_controller.rb

```diff
@@ -1,17 +1,22 @@
 class RentsController < ApplicationController
   include Wor::Paginate
-
   def create
-    @rent = Rent.new(params[:rent])
+    @rent = Rent.new(rent_params)
     if @rent.save
-      render json: @rent, status: :created
+      render json: @rent
```

Comment:
HTTP status codes are really important for API REST (https://en.wikipedia.org/wiki/Representational_state_transfer). So we do want to point out the status of the response. If the object has been created, then we need to return the status `:created`(201).

For more information you can read: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success